docker-deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
```

Comment:
What's the benefit of having it build in the script step rather than in this bash script?